src/js/main.js

```diff
@@ -153,7 +161,25 @@ function fuzzySearch(){
         populateSearchSuggestions(searchString);
         var fuzz = new Fuse(suggestionList, options);
         var fuzzResult = fuzz.search(searchString);
-        populateSuggestionsBox(fuzzResult);
+        const highlightedResult = fuzzResult.map(({ item, matches }) => {
```

Comment:
I am following the airbnb linting conventions there, without spaces it is harder to see when the parameters start and end, sure about it?